openupgrade_scripts/scripts/fleet/14.0.0.1/pre-migration.py

```diff
@@ -9,3 +9,6 @@ def migrate(env, version):
         env.cr,
         {"fleet_vehicle_log_contract": [("state", None, None), ("name", None, None)]},
     )
+    openupgrade.remove_tables_fks(
```

Comment:
Why is this needed?

Reply:
because... obsolete tables, always better to remove the fks